lib/open_food_network/feature_toggle.rb

```diff
@@ -51,7 +51,7 @@ module FeatureToggle
       # Copy features here that were activated in a migration so that new
       # instances, development and test environments have the feature active.
       "admin_style_v3" => <<~DESC,
-        Test the work-in-progress design updates.
+        Test the work-in-progress design updates. Activated for admins, new users, and 75% of all users.
```

Comment:
Nice idea to keep the description updated 👍 

However, these details are not hard coded and can be manipulated in the admin interface according to specific needs (see below). So they could be wrong, and cause confusion. Can you please remove this change?

See http://localhost:3000/admin/feature-toggle/features/admin_style_v3
![Screenshot 2024-08-12 at 5 42 51 pm](https://github.com/user-attachments/assets/f2654461-5b69-4ceb-9577-24ae66c5ec52)

Reply:
I just realised I can do that easily now.
```suggestion
        Test the work-in-progress design updates.
```